libs/shared/src/consts/providers/provider.enum.ts

```diff
@@ -69,6 +69,7 @@ export enum SmsProviderIdEnum {
   Termii = 'termii',
   AfricasTalking = 'africas-talking',
   Novu = 'novu-sms',
+  Kavenegar = 'kavenegar-novu-sms',
```

Comment:
```suggestion
  Kavenegar = 'kavenegar',
```
They do not seem to offer other products than SMS so we don't need to differentiate it.